fly.toml

```
@@ -0,0 +1,13 @@
+app = "api-mock-server"
+primary_region = "sin"
+
+[build]
+  dockerfile = "Dockerfile.mock"
```

Regarding previous comment, we should change here as well:
```suggestion
  dockerfile = "Dockerfile"
```